_posts/2023-07-31-workshops.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Online Training Workshop Series
+---
+
+MDAnalysis is excited to be kicking off a series of free online training workshops in Fall 2023. The first will be held on **October 25, 2023 from 15:00-18:30 UTC**. The next workshop will be organized for Australasian time zones and will be announced shortly.
+
+This remote workshop will consist of an introduction to the MDAnalysis Python library, as well as an opportunity for hands-on learning through basic MDAnalysis tutorials. The workshop will be delivered to a small group to allow interactive discussions, questions, and participant engagement. It is designed for beginners to MDAnalysis who already have a working knowledge of Python and working with shell and notebook environments.
+
+# Register soon (places limited)
+If you are interested in participating in this workshop, fill out [this short application][eventbrite] form before **August 30, 2023**. Selected participants will be notified by September 27. 
```

Comment:
as discussed: move deadline to middle of September and notification to early October